client/src/app/pages/issues/issue-detail-drawer/path-display.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+interface PathDisplayProps {
+  path: string;
+}
+
+const PathDisplay: React.FC<PathDisplayProps> = ({ path }) => {
+  const formattedPath = path
+    .split("/")
```

Comment:
My only concern here is if the path will ever has a windows based backslash `\` separator.  I don't know if that is even something that will happen...